dev/test/query.ts

```diff
@@ -517,7 +517,8 @@ describe('query interface', () => {
       },
     };
 
-    return createInstance(overrides).then(firestore => {
+    return createInstance(overrides).then(firestoreInstance => {
+      firestore = firestoreInstance;
```

Comment:
Could you please share with me what's the difference between this change and the old implementation?

Reply:
There are the `beforeEach` and `afterEach` functions that respectively assign vanilla firestore instance to `firestore` variable declared outside of `it` test, and then calling `verifyInstance` on `firestore` variable after test has run.

There were 2 problems:

1. The `firestore` variable was never set as part of test when creating a instance with overrides.
2. The `verifyInstance` is an asynchronous function, and the `afterEach` function did not wait for it to complete, therefore any error would be lost.

It is important to assign value to `firestore` variable outside of test in order for cleanup in `afterEach` to verify executors are finished in firestore state.